project/Dependencies.scala

```diff
@@ -81,6 +83,7 @@ object Dependencies {
         Compile.akkaHttp,
         Compile.akkaHttp2Support,
         Compile.akkaDiscovery,
+        Compile.akkaHttpCors % "provided",
```

Comment:
Hmm I'm not sure if this'll work, since the ServiceHandler class is used for native gRPC as well and might not be loadable if it refers to classes that are not on the classpath. Perhaps neat to move that to its own class anyway?

Reply:
Has been split in your latest commits.